lutris/runners/atari800.py

```diff
@@ -105,8 +105,8 @@ def find_good_bioses(self, bios_path):
         good_bios = {}
         for filename in os.listdir(bios_path):
             real_hash = system.get_md5_hash(os.path.join(bios_path, filename))
-            for bios_file in self.bios_checksums.keys():
-                if real_hash == self.bios_checksums[bios_file]:
+            for bios_file, checksum in self.bios_checksums:
```

Comment:
You can't iterate dictionaries like that, you need .items()

Reply:
Duh, holy moly my bad. Sorry. Fixed both loops where I made this grave mistake. Sorry for the inconvenience!